src/cascadia/TerminalControl/ControlCore.cpp

```diff
@@ -1751,7 +1751,7 @@ namespace winrt::Microsoft::Terminal::Control::implementation
     {
         const auto lock = _terminal->LockForWriting();
         _terminal->SetSearchHighlights({});
-        _terminal->SetSearchHighlightFocused({});
+        _terminal->SetSearchHighlightFocused({}, _searchScrollOffset);

```

Comment:
I suppose you can just pass 0 instead of `_searchScrollOffset` here?

Reply:
updated